### PR TITLE
Merchant dashboard us34

### PR DESCRIPTION
**Pull request by LHJE**
[Reference story or milestone being closed]

**Contributors**
- Driver(s): @LHJE 
- Navigator(s): @priyapower 
  
**What was changed:**
- We added Merchant information to an employee's merchant dashboard.

**Any tech notes (if applicable)**

### CODE REVIEW

File: app/controllers/merchant/dashboard_controller.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@ class Merchant::DashboardController < Merchant::BaseController
   before_action :require_merchant
 
   def index
+    @merchant = Merchant.find(current_user.merchant_id)
```
Comment (priyapower) [inline app/controllers/merchant/dashboard_controller.rb:5]: This makes sense currently - may need updates as we continue to update this Milestone

File: app/views/merchant/dashboard/index.html.erb [hunk 2]
```diff
@@ -1 +1,14 @@
-#merchants
+
+
+<h1>My Employer</h1>
+
+<section id = "merchant-info">
```
Comment (priyapower) [inline app/views/merchant/dashboard/index.html.erb:5]: Looks like a great initial View Page

File: spec/features/merchant/show_spec.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe 'Site Navigation' do
+  describe 'As a Merchant' do
+    before :each do
+      @mike = Merchant.create!(name: "Mike's Print Shop", address: '123 Paper Rd.', city: 'Denver', state: 'CO', zip: 80203)
+      @mikes_employee = User.create!(name:"Leah", address:"123 Sesame Street", city:"New York", state:"NY", zip:"90210", email: "Leahsocool@gmail.com", password:"Imeanit", password_confirmation:"Imeanit", role: 1, merchant_id: @mike.id)
```
Comment (priyapower) [inline spec/features/merchant/show_spec.rb:7]: Aahh... THIS is how we are connecting merchant users to merchants. I still wonder where that officially happens? Is that an admin privilege?

File: spec/features/merchant/show_spec.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+    it "can see the name and full address of the merchant I work for" do
+      expect(page).to have_content(@mike.name)
+      expect(page).to have_content(@mike.address)
+      expect(page).to have_content(@mike.city)
+      expect(page).to have_content(@mike.state)
+      expect(page).to have_content(@mike.zip)
+    end
+
+    it "if an employee logs in accidentally as a regular user, they cannot see the merchant dashboard, and must log back in as a merchant"
```
Comment (priyapower) [inline spec/features/merchant/show_spec.rb:22]: Awesome edge case or sad path pending test - lets make sure to write this in BEFORE the milestone PR

File: spec/features/navigation/merchant_nav_spec.rb [hunk 5]
```diff
@@ -5,15 +5,9 @@
     before :each do
       @merchant = User.create!(name:"Leah", address:"123 Sesame Street", city:"New York", state:"NY", zip:"90210", email: "Leahsocool@gmail.com", password:"Imeanit", password_confirmation:"Imeanit", role: 1)
 
-      visit "/"
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@merchant)
```
Comment (priyapower) [inline spec/features/navigation/merchant_nav_spec.rb:8]: WOO! What a better way to "login" during a test

**Review by priyapower — COMMENTED** [review]
Looks great!
